### DISCUSSION TITLE
deepin-terminal将远程连接的分组设置在一级入口是否能够提升体验？

### DISCUSSION THREAD

**Discussion by liwl1991** [linuxdeepin/developer-center#4066] [Features Request & Ideas | 特性请求 & 头脑风暴]
类似：https://bbs.deepin.org/post/243700
在使用deepin-terminal的时候，远程管理作为其组件，能够很好地支撑非本地节点连接管理。
假如用户有很多节点需要管理，必然会有大量的远程管理标签。这些标签实际上是有分组的，但是被隐藏在【高级设置】里面，这个很难被用户发现，并且将远程节点分组管理，也是一个很常规常用的管理操作。
产品这边是否考虑一下，根据用户使用习惯，优化一下【分组】的入口。放置在【证书】下，或者【服务器】名下，非必需
代码改动上很简单，我自己已经通过修改代码实现

**Comment by liwl1991** [comment]
@ArchieMeng @meiyixiang

**Comment by meiyixiang** [comment]
这个建议很好，组是一个逻辑概念，应该是高于具体的服务器的，现在这种组在服务器内添加的方式不符合使用逻辑，最近在做V23的集成发布工作，下个版本会吧这个需求进行优化。

**Comment by ArchieMeng** [comment]
另外，关于分组这个选项，如果能输入时候联想已有分组以供选择，我觉得也会更方便些．可以同时满足选择已有分组和添加新分组的需求．

**Comment by meiyixiang** [comment]
这个优化的时候会整体考虑新增、已有分组创建、移动等

**Reply by liwl1991** [reply]
赞！期待v23的新终端

**Comment by meiyixiang** [comment]
最近也做了一些新的特性主要在于提升易用性，有一些是开发者和入的。
